packages/composites/outlier-detection/README.md

```diff
@@ -9,74 +9,134 @@ If all checks passes, the adapter returns the median from the source data provid
 
 The adapter takes the following environment variables:
 
-- `SOURCE_ADAPTERS`: Required list of data source adapters. One or multiple of: `xbto|genesisvolatility|dxfeed`
-- `CHECK_ADAPTERS`: List of adapters to check against. One or multiple of: `deribit|oilpriceapi|dxfeed`
-- `CHECK_THRESHOLD`: Set a percentage deviation threshold against the check data sources. Set to 0 or empty to not
-perform this check.
-- `ONCHAIN_THRESHOLD`: Set a percentage deviation threshold against the on-chain value. Set to 0 or empty to not
-perform this check.
-- `RPC_URL`: ETH RPC URL to read the reference data value. Required by runlog requests.
+1. A data source adapter
 
-This adapter also takes any configuration variables of the underlying adapters. Simply prefix the environment variables.
-E.g. dxFeed endpoint URL:
+   | Required? |                  Name                  |                      Description                      | Options | Defaults to |
+   | :-------: | :------------------------------------: | :---------------------------------------------------: | :-----: | :---------: |
+   |           |        `XBTO_DATA_PROVIDER_URL`        |        The location of a XBTO external adapter        |         |             |
+   |           | `GENESIS_VOLATILITY_DATA_PROVIDER_URL` | The location of a Genesis Volatility external adapter |         |             |
+   |           |       `DXFEED_DATA_PROVIDER_URL`       |       The location of a DXFeed external adapter       |         |             |
```

Comment:
I think this comment applies to the whole PR and not just this, but I think we should remove any specific references to underlying adapters (except examples in the Readme).

This adapter is essentially just checking one group of adapters against a different group of adapters and/or the on-chain value, and we should be able to extend that to use any underlying adapters.

Reply:
Interesting idea, yeah that makes a lot of sense.

So checking the on-chain value would become more optional then right?

Reply:
Yes – checking the on-chain value should be optional. The reason for fetching it (either from meta data or from RPC) is that the adapter should return the on-chain value if outlier detection is triggered.